www_files/app.wsgi

```diff
@@ -0,0 +1,59 @@
+import importlib.util
+import sys
+import os
+
+MODULE_PATH = "/hpc/users/wallad07/www/mmeds-meta/mmeds/__init__.py"
+# MODULE_PATH = "/home/david/Work/mmeds-meta/mmeds/__init__.py"
```

Comment:
cleanup?

Reply:
Depending on if I'm local or on minerva I need to change that value